spec.md

```diff
@@ -293,3 +293,74 @@ MaritalStatus
   }
 }
 ```
+
+
+## PrivateUnsecuredLoanOffering
+
+This event pertains to offers for unsecured loans for Swedish customers.
+
+Event (root element)
+
+| Name            | C.   | Type        | V. | Remark                          |
+|-----------------|------|-------------|----|---------------------------------|
+| offer           | 1    | Application | v0 | Contains the offer for loan     |
+| broker          | 1    | reverse-dns | v0 | Domain-name of the broker       |
+| brokerReference | 1    | string      | v0 | Per-broker unique, not globally |
+
+Offer
+
+| Name                  | C.   | Type               | V. | Remark                                                                         |
+|-----------------------|------|--------------------|----|--------------------------------------------------------------------------------|
+| effectiveInterestRate | 1    | string             | v0 | A number formatted as a string to reduce risk of rounding error                |
+| nominalInterestRate   | 1    | string             | v0 | A number formatted as a string to reduce risk of rounding error                |
+| minOfferedCredit      | 1    | number             | v0 | The smallest amount that can be offered at the current nominal interest rate * |
+| offeredCredit         | 1    | number             | v0 | Offered amount, this is the value that the APR is based on.                    |
+| maxOfferedCredit      | 1    | number             | v0 | The largest amount that can be offered at the current nominal interest rate *  |
+| establishingFee       | 1    | number             | v0 | The initial payment for establishment of the loan                              |
+| termFee               | 1    | number             | v0 | The invoice cost or other fixed fees to be paid along with amortisation and interest every month |
```

Comment:
remove the word invoice cost as this assumes that is for the invoice that might be avoidable.
remove `The invoice cost or other` the rest of the sentence is very good